src/libs/actions/IOU.ts

```diff
@@ -1858,23 +1858,40 @@ function buildOnyxDataForInvoice(
     return [optimisticData, successData, failureData];
 }
 
+type BuildOnyxDataForTrackExpenseParams = {
+    chatReport: OnyxInputValue<OnyxTypes.Report>;
+    iouReport: OnyxInputValue<OnyxTypes.Report>;
+    transaction: OnyxTypes.Transaction;
+    iouCreatedAction: OptimisticCreatedReportAction;
+    iouAction: OptimisticIOUReportAction;
+    reportPreviewAction: OnyxInputValue<ReportAction>;
+    transactionThreadReport: OptimisticChatReport | null;
+    transactionThreadCreatedReportAction: OptimisticCreatedReportAction | null;
+    shouldCreateNewMoneyRequestReport: boolean;
+    policy?: OnyxInputValue<OnyxTypes.Policy>;
+    policyTagList?: OnyxInputValue<OnyxTypes.PolicyTagLists>;
+    policyCategories?: OnyxInputValue<OnyxTypes.PolicyCategories>;
+    existingTransactionThreadReportID?: string;
+    actionableTrackExpenseWhisper?: OnyxInputValue<OnyxTypes.ReportAction>;
+};
```

Comment:
Would you please break this into sub-objects like we do for MoneyRequestOptimisticParams?

For example we can group together the chat params, iou, policy.

https://github.com/Expensify/App/blob/ac0ba20ed5836d900238d7e49e2ca74c0c3747cf/src/libs/actions/IOU.ts#L393-L403